src/main/java/at/hannibal2/skyhanni/data/jsonobjects/repo/DisabledApiJson.kt

```diff
@@ -0,0 +1,9 @@
+package at.hannibal2.skyhanni.data.jsonobjects.repo
+
+import com.google.gson.annotations.Expose
+
+data class DisabledApiJson(
+    @Expose val disabledBazaar: Boolean,
+    @Expose val disabledMoulberryLowestBin: Boolean,
+    @Expose val disabledNpcPrices: Boolean,
```

Comment:
rename npc prices to `disableHypixelItems` since this not only impacts npc prices, but also mote prices and base stats.